components/json/rs/src/lib.rs

```diff
@@ -0,0 +1,73 @@
+#[cxx::bridge(namespace = json)]
+mod ffi {
+    extern "Rust" {
+      fn convert_i64_to_string(path: &str, json: &str) -> String;
+      fn convert_string_to_i64(path: &str, json: &str) -> String;
+    }
+}
+
+// Converts int64/uint64 value to string and serializes json with new values.
+// A path is a Unicode string with the reference tokens separated by /. Inside tokens / is replaced by ~1 and ~ is replaced by ~0.
+// Examples: convert_i64_to_string("/a/b", json) for { a : { b : 1 }}
+//           convert_i64_to_string("/a/0", json) for { a : [ 1 ]}
+pub fn convert_i64_to_string(path: &str, json: &str) -> String {
+  let mut unwrapped_value: serde_json::Value = serde_json::from_str(&json).unwrap_or_else(|e| {
+    eprintln!("Failed to parse JSON: {}", e);
```

Comment:
I believe we are using a particular logging library to proxy to base/logging cc @evq

Reply:
this was removed with latest changes